src/components/MessagesList/MessagesList.vue

```diff
@@ -643,8 +710,8 @@ export default {
 			}
 		},
 
-		handleMessageEdited(id) {
-			this.messagesGroupedByAuthor = this.prepareMessagesGroups(this.messagesList)
+		handleMessageEdited() {
+			this.messagesGroupedByDateByAuthor = this.prepareMessagesGroups(this.messagesList)
```

Comment:
This one should be a soft update -> we're splitting a single author group into 2-3 groups